src/Stripe/Services/Discounts/StripeDiscountService.cs

```diff
@@ -0,0 +1,37 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Stripe.Services.Discounts {
```

Comment:
the namespace should always just be "Stripe" - I do this so users don't have to include a bunch of namespaces. They just always include Stripe and they are done.